bin/ocaml/ocaml_merlin.ml

```diff
@@ -264,6 +291,14 @@ module Dump_dot_merlin = struct
             ~doc:
               "The path to the folder of which the configuration should be printed. \
                Defaults to the current directory.")
+    and+ ctx_name =
+      Arg.(
+        value
+        & opt (some string) None
+        & info
+            [ "context" ]
+            ~docv:"CONTEXT"
+            ~doc:"The Dune context in which the command will return information for")
```

Comment:
What do you think about this phrasing? I can't exactly point out what, but the current sentence reads sort of weird.

```suggestion
            ~doc:"Dune will return information for this build context")
```